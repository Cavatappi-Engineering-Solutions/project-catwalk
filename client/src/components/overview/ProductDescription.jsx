import React from 'react'

const ProductDescription = ({ product }) => (
  <div className='product_slogan_description'>
    <section>
      <h3 className='product_slogan'>{product.slogan}</h3>
      <p className='product_description'>{product.description}</p>
    </section>
  </div>
)

export default ProductDescription
