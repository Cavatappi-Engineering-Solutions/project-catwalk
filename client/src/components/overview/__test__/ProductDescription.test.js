import React from 'react'
import renderer from 'react-test-renderer'
import ProductDescription from '../ProductDescription.jsx'

it('renders correctly', () => {
  const tree = renderer.create(<ProductDescription
    product={{
      campus: 'hrnyc',
      category: 'Pants',
      created_at: '2021-01-12T21:17:59.200Z',
      default_price: '40.00',
      description: "Whether you're a morning person or not.  Whether you're gym bound or not.  Everyone looks good in joggers.",
      features: [{}, {}],
      id: 11003,
      name: 'Morning Joggers',
      slogan: 'Make yourself a morning person',
      updated_at: '2021-01-12T21:17:59.200Z'
    }}
  />).toJSON()
  expect(tree).toMatchSnapshot()
})
