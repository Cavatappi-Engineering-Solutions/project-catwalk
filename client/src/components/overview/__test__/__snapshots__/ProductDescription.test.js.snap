// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="product_slogan_description"
>
  <p
    className="product_slogan"
  >
    Make yourself a morning person
  </p>
  <p
    className="product_description"
  >
    Whether you're a morning person or not.  Whether you're gym bound or not.  Everyone looks good in joggers.
  </p>
</section>
`;
